.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: c
+dist: trusty
+
+cache:
+  directories:
+    - ~/docker
+
+matrix:
+  include:
+  - env: BASE=ubuntu IMAGE=ubuntu:16.04 CCO=gcc
+
+  - env: BASE=ubuntu IMAGE=ubuntu:16.04 CCO=clang
+
+  - env: BASE=ubuntu IMAGE=ubuntu:16.10 CCO=gcc
+
+  - env: BASE=fedora IMAGE=fedora:25    CCO=gcc
+
+before_install:
+  - cd ..
+  - git clone --depth 1 https://github.com/GNOME/jhbuild.git
+
+before_script:
+  - export NAME=$(echo $IMAGE | sed "s/:/_/g")
+  - ls -lah ~/docker
+
+  # Check if there is a saved base image
+  - if [[ -f ~/docker/$NAME.tar.gz ]]; then cat ~/docker/$NAME.tar.gz | docker import - local/$NAME; fi
```

Comment:
Because of import syntax (it forces you to use file OR tag, never both). 
- cat is "the way " docker docs uses it.

Should I change it to redirection?